Common/cpp/SharedItems/ShareableValue.cpp

```diff
@@ -239,8 +250,10 @@ jsi::Value ShareableValue::toJSValue(jsi::Runtime &rt) {
     }
     case ValueType::HostFunctionType: {
       auto hostFunctionWrapper = ValueWrapper::asHostFunctionWrapper(valueContainer);
-      if (hostFunctionWrapper->hostRuntime == &rt) {
+      auto hostRuntime = hostFunctionWrapper->value->hostRuntime;
```

Comment:
I think we can use here `auto &` instead of `auto` to avoid unnecessary copying of objects.